src/open_inwoner/components/templates/components/Form/FileInput.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n form_tags button_tags icon_tags %}
+
+<div class="form__control {{ extra_classes|default:"upload" }}">
+    <div id="inputfile-group" class="inputfile-group">
```

Comment:
@vaszig I've done a bit of research on naming conventions, since we don't have one standard yet in this project.
I found it's not _technically_ necessary, but 'good practice' to have different names for ID's and classes in HTML.
Plus: I will set ID-names with an underscore from now on, and classes with a dash (because for classes the dash can be used as a modifier). I was using camelCase for some ID names, because of javascript, but that's not useful for CSS.